spec/fixtures/mock_cdn_repo_remote/all_pods.txt

```diff
@@ -0,0 +1,2 @@
+BeaconKit
+SDWebImage
```

Comment:
will this file grow to be too large? not this spec specifically but on trunk. wondering if it should be split up

Reply:
[It's just 800KB now](
https://github.com/igor-makarov/Specs/blob/master/all_pods.txt). 

In fact, similar to Bundler's compact index, this saves time by coalescing it into jist one CDN call.

By the way, this only gets used in searching. If you specify `pod 'DfPodTest'` it will optimistically go for the specific pod index.

Reply:
👍